api/envoy/config/filter/http/ext_authz/v2/ext_authz.proto

```diff
@@ -24,7 +24,7 @@ option (udpa.annotations.file_status).package_version_status = FROZEN;
 // External Authorization :ref:`configuration overview <config_http_filters_ext_authz>`.
 // [#extension: envoy.filters.http.ext_authz]
 
-// [#next-free-field: 11]
+// [#next-free-field: 12]
```

Comment:
@fpliu233 let's chat internally about the need for v2 here. We have technically frozen the v2 API, so we can only add things like this if we want to make an exception, I'd like to understand if we should be making that case.